build-tests/heft-webpack5-everything-test/src/chunks/image.d.png.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+declare const path: string;
+
+export default path;
```

Comment:
Interesting this is getting checked in instead of produced as part of the build. Is this because we don't have a tool that can produce these assets checked into Rushstack?

Reply:
Yeah, we don't have a static-asset-typings-plugin in rushstack.